app/mailers/notification_mailers/contacts_birthday.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module NotificationMailers
+  class ContactsBirthday < NotificationMailers::Base
+    attr_accessor :contact
+
+    def set_headers(contact_id)
```

Comment:
Do not prefix writer method names with `set_`.

Reply:
As stated above, we think this should stay. By the way, this PR is ready for review :)